eea/facetednavigation/widgets/sorting/edit.js

```diff
@@ -21,7 +21,7 @@ FacetedEdit.SortingWidget = function(wid, context){
 FacetedEdit.SortingWidget.prototype = {
   set_default: function(element){
     var value = this.select.val();
-    if(this.reverse.attr('checked')){
+    if(this.reverse.is(':checked')){
```

Comment:
You can use this.reverse.prop('checked') too.
See the difference between attr and prop here: http://api.jquery.com/prop/